CoreGraphFile/src/au/gov/asd/tac/constellation/graph/file/io/GraphFileConstants.java

```diff
@@ -28,6 +28,7 @@
      * No constructor.
      */
     private GraphFileConstants() {
+        throw new IllegalStateException("Utility Class");
```

Comment:
Would change the message to "Constants class"

Reply:
Done